content/docs-beta/SUSHI/migration/_index.md

```diff
@@ -5,42 +5,91 @@ weight: 30
-#### 4 - Remove fsh/ Directory
-At this point, all of the contents of the **customized-ig/fsh** folder should have been moved out of **customized-ig/fsh**, so that folder can be safely deleted.
-#### 5 - Update Content of sushi-config.yaml
-Update **sushi-config.yaml** to remove the `template` property if necessary. The `template` property is used by SUSHI to generate an **ig.ini** file, and its usage is now deprecated. The **ig.ini** file must be managed directly. If SUSHI detects usage of the `template` property to generate an **ig.ini** file, it will log an error message containing the **ig.ini** that would be generated by that `template` property. Copy these contents, and place them in an **ig.ini** file located at **customized-ig/ig.ini**.
+The SUSHI project structure and configuration changed significantly between pre-1.0.0 versions of SUSHI and versions of SUSHI 1.0.0 or greater. This guide will assume you currently have a project structure similar to the **customized ig** shown in the [project structure documentation](/docs/sushi/project/#ig-projects) for pre-1.0.0 versions of SUSHI.
+
+To migrate your project, follow these steps:
+
+### 0 - Create a Backup
+
+Before beginning the migration process, **make a copy of your entire project directory structure**, and keep it in a safe place.
+
+### 1 - Delete SUSHI-Generated Files
+
+If you have previously used SUSHI, you have SUSHI-generated files that need to be cleaned up. If you use Git, you can remove all untracked files using the command:
+
+```shell
+{{< terminal >}} git clean -xfd
+```
+
+Before executing this command, you can run `git clean -xfdn` to preview the list of files git will remove. If it looks OK, then run the command above.
+
+You can refer to **./SUSHI-GENERATED-FILES.md** for a partial list of files to delete. (This list does not include generated profiles, value sets, extensions, and examples found in the **./input** directory that should also be deleted.)
+
+Keep in mind that if you created FHIR artifacts NOT using SUSHI, those files must be retained. Typically, they will be found under the **./input** directory.
+
+
+### 2 - Rearrange Directories
+
+Move the entire contents of **./fsh/ig-data** directory to the top level (do not move the **/ig-data** directory, just its contents). 
+
+> **NOTE:** If after the cleanup, you still have a top-level **./input** directory, then _merge_ the contents of **./fsh/ig-data/input** into that directory (rather than replacing it).
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}}
If after the cleanup, you still have a top-level **./input** directory, then _merge_ the contents of **./fsh/ig-data/input** into that directory (rather than replacing it).
{{% /alert %}}
```

The above approach is more consistent with how notes are formatted across FSHSchool (and in this guide).

Reply:
got it.